LICENSE

```diff
@@ -45,3 +45,34 @@ BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
 PARTICULAR PURPOSE. THE SOFTWARE PROVIDED HEREUNDER IS ON AN "AS IS" BASIS, AND
 THE UNIVERSITY OF CALIFORNIA HAS NO OBLIGATIONS TO PROVIDE MAINTENANCE, SUPPORT,
 UPDATES, ENHANCEMENTS, OR MODIFICATIONS.
+
+__________________________________________________________________________
+This software contains code from the cl-json project in the following file:
+/postmodern/json-encoder.lisp
```

Comment:
Why don't we ASDF-depend on `cl-json` instead of copying code from it into this repository?

Reply:
Phoe, thank you for the review!

The escapes seem to have been removed by github. I did type them in. Will double check the tests though. 

Good question about whether we should asdf cl-json and include it all when we only need a piece. My original thought was why include a library when users may use a different json library. Quite happy to pull it back out.

Reply:
re: escapes: oh, I see. You need to explicitly use code blocks (\`\`\`) I think, otherwise GitHub might eat the backslashes.

I don't know about the JSON library; I'd suggest letting other people chip in about what would be the best way forwards.